### DISCUSSION TITLE
Docker Deployment works but PDF give "Failed to load PDF file".

### DISCUSSION THREAD

**Discussion by davidfcunningham** [OpenSignLabs/OpenSign#1128] [Q&A] [answered]
Hello,

The Docker deployment seems to work perfectly...  but any PDF I upload gives "Failed to load PDF file." as an error.  Any advice is appreciated!

**Comment by andrew-opensignlabs** [comment, marked as answer]
This happens primarily when CORS settings are not properly updated.  You may find errors like below in your browser console -
Access to fetch at 'https://foo.nyc3.digitaloceanspaces.com/exported_file_4627_0000-00-00T00%3A45%3A43.344Z.pdf'
from origin 'http://localhost:3000' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header
is present on the requested resource. If an opaque response serves your needs, set the request's mode to
'no-cors' to fetch the resource with CORS disabled.

You can find more details about configuring CORS below -

[How to Configure CORS on DigitalOcean Spaces](https://docs.digitalocean.com/products/spaces/how-to/configure-cors/)
[Configuring cross-origin resource sharing on AWS S3](https://docs.aws.amazon.com/AmazonS3/latest/userguide/enabling-cors-examples.html)
